backend/src/apiserver/resource/client_manager_fake.go

```diff
@@ -90,6 +93,16 @@ func NewFakeClientManagerOrFatal(time util.TimeInterface) *FakeClientManager {
 	return fakeStore
 }
 
+func NewFakeClientManagerOrFatal_KFAM_Authorized(time util.TimeInterface) *FakeClientManager {
```

Comment:
This is too specific to a certain client. 
Here is an example of how to intercept the behavior of a fake class.
https://github.com/kubeflow/pipelines/blob/1b85a614bf812ce18bf52800568ea39b7fe5f645/backend/src/apiserver/resource/workflow_fake.go#L146

Reply:
This was used to create a fakeclientmanager with the unauthorized kfamclient. Creating a new FakeAuthorizedClientManager would be lots of duplicate codes to implement the ClientManagerInterface functions. Is that what you are suggesting?

Reply:
The code refers to FakeBadWorkflowClient which will be a different fake client with different behavior. That's a better way if you want to introduce a different behavior for a fake client.

Reply:
the resourcemanager cannot access an unexported field, do you have any suggestions?

Reply:
Done